### PR TITLE
Remove `anyhow` dependency

### PR DESCRIPTION
**Pull request by Kijewski**

### CODE REVIEW

File: gtk3-macros/src/attribute_parser.rs [hunk 1]
```diff
@@ -45,13 +44,31 @@ impl Parse for TemplateSource {
...
-    let attr = match input.attrs.iter().find(|a| a.path().is_ident("template")) {
-        Some(attr) => attr,
-        _ => bail!("Missing 'template' attribute"),
-    };
+#[derive(Debug)]
+pub enum ParseTemplateSourceError {
+    MissingAttribute,
+    Parse(syn::Error),
+}
+
```
Comment (bilelmoussaoui) [inline gtk3-macros/src/attribute_parser.rs:52]: Please implement std::error::Error as well.

Comment (Kijewski) [reply gtk3-macros/src/attribute_parser.rs:52]: Implemented.

**Review by bilelmoussaoui — APPROVED** [review]

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
For the CI failure, we'll need to wait for https://github.com/gtk-rs/gtk3-rs/pull/831 to be merged first.

**Comment by Kijewski** [comment]
Rebased, but nasm is down.

**Comment by GuillaumeGomez** [comment]
Thanks! Gonna cherry-pick it for the new release too.

**Pull request merged**